src/bin/test_userproc_even.c

```diff
@@ -0,0 +1,29 @@
+int main();
+int _start() {
+    return main();
+
+    // invoke system call that kills the process
+}
+
+#include "../syscall.h"
```

Comment:
The customary way to solve this is to set up a directory that contains the "public" includes, then tell the compiler to make it part of the include path with the `-I` option.  That way, user-level code doesn't need to know anything about the filesystem layout, it just accepts what the compiling environment has set up.